include/structs.h

```diff
@@ -1217,6 +1227,7 @@ typedef struct Object_64 {
         Object_Bridge_WhaleRamp bridge_whale_ramp;
         Object_80011AD0 obj80011AD0;
         Object_8001B7A8 obj8001B7A8;
+        Object_80021400 obj80021400;
```

Comment:
These are supposed to be a union of all the unk64 structs. Should this instead be a `Object_64_80021400`? Or does the unk64 struct itself contain a struct pointer at offset 0x64?

Reply:
Gotcha. In that case can you rename `Object_64_80021400` to `Object_80021400_64`? That way it's not confused as an Object64, and the left-to-right nature of reading implies the structure of the type.